Logger/FileLogger.cs

```diff
@@ -0,0 +1,16 @@
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public string filePath {get; set;}
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            StreamWriter LogFile = File.AppendText(filePath);
```

Comment:
Another way to accomplish this is with the [using ](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement) statement.
```suggestion
            using(streamWriter streamWriter = new StreamWriter(filePath, true))
            {
                 streamWriter.WriteLine("Log message stuff here...");
            }
```